.github/ISSUE_TEMPLATE/add_dataset.yml

```
@@ -0,0 +1,122 @@
+      - Creative Commons Attribution Non Commercial 2.0 Generic
+      - Creative Commons Attribution Non Commercial 2.5 Generic
+      - Creative Commons Attribution Non Commercial 3.0 Unported
+      - Creative Commons Attribution Non Commercial 4.0 International
+      - Creative Commons Attribution Non Commercial No Derivatives 1.0 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 2.0 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 2.5 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 3.0 Unported
+      - Creative Commons Attribution Non Commercial No Derivatives 3.0 IGO
+      - Creative Commons Attribution Non Commercial No Derivatives 4.0 International
+      - Creative Commons Attribution Non Commercial Share Alike 1.0 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 2.0 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 2.5 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 3.0 Unported
+      - Creative Commons Attribution Non Commercial Share Alike 4.0 International
+      - Creative Commons Attribution No Derivatives 1.0 Generic
+      - Creative Commons Attribution No Derivatives 2.0 Generic
+      - Creative Commons Attribution No Derivatives 2.5 Generic
+      - Creative Commons Attribution No Derivatives 3.0 Unported
+      - Creative Commons Attribution No Derivatives 4.0 International
+      - Creative Commons Attribution Share Alike 1.0 Generic
+      - Creative Commons Attribution Share Alike 2.0 Generic
+      - Creative Commons Attribution Share Alike 2.0 England and Wales
+      - Creative Commons Attribution Share Alike 2.5 Generic
+      - Creative Commons Attribution Share Alike 3.0 Unported
+      - Creative Commons Attribution-Share Alike 3.0 Austria
+      - Creative Commons Attribution Share Alike 4.0 International
+      - Creative Commons Public Domain Dedication and Certification
+      - Creative Commons Zero v1.0 Universal
+      - OCLC Research Public License 2.0
+      - Open Government Licence v1.0
+      - Open Government Licence v2.0
+      - Open Government Licence v3.0
+  - type: input
+    id: other-licence
+    attributes:
+      label: Other licence 
+      description:  Other licence if it is not included in the above dropdown list
+    validations:
+      required: false
+  - type: dropdown
+    id: dataset-access
+    attributes:
+      label: How can you access this data
+      description: How can one access this data?
+      options:
+        - As a download from a repository/website 
+        - Via an open API
+        - Via an API that requires free registration 
+        - Other
+    validations:
+      required: true
+  - type: checkboxes
+    id: confirm-open
+    attributes:
+      label: Confirm the dataset is open
+      description: Please confirm the dataset is currently openly available 
```

Maybe to avoid ambiguity with the word "open"...
```suggestion
      label: Confirm the dataset has an open licence
      description: Please confirm the dataset is currently available under an open licence
```